src/mlpack/methods/lsh/lsh_search.hpp

```diff
@@ -159,8 +163,6 @@ class LSHSearch
 
   //! Get the number of projections.
   size_t NumProjections() const { return projections.size(); }
-  //! Get the projection matrix of the given table.
-  const arma::mat& Projection(const size_t i) const { return projections[i]; }
```

Comment:
We should mention the removal of this function in HISTORY.txt, and mention that users should now use `Projections().slice(i)` instead.